### DISCUSSION TITLE
No podemos crear la imagen docker.

### DISCUSSION THREAD

**Discussion by xavier-ramentol** [ciberado/azure-intro-ed01#5] [sesión 03]
al ejecutar el comando docker build -t helloworld:basic . no sale lo siguiente (hemos verificado el contenido de los archivos y todo OK)

![image](https://github.com/user-attachments/assets/b56e4ef4-e2a0-4065-a1a1-0646d632c08d)

**Comment by xavier-ramentol** [comment]
tema solucionado. hay que eliminar todos los comentarios del fichero Dockerfile (algún carácter extraño???)

**Comment by ciberado** [comment]
Muchas gracias! Reviso el caso, a ver si entiendo por qué :)

**Comment by ciberado** [comment]
Nada, no consigo reproducirlo :/ He movido los comentarios a principios de línea, por si eso ayuda a editar el fichero. Merci otra vez!
